src/main/java/core/basesyntax/strategy/operation/BalanceOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.operation;
+
+public class BalanceOperationHandler implements OperationHandler {
+    @Override
+    public Integer updateBalance(Integer balance, Integer value) {
+        if (value < 0) {
+            throw new RuntimeException(String.format("Buyers will not be able to buy %s units. "
+                    + "%s is incorrect input.", value, value));
+        }
+        return balance + value;
```

Comment:
You are using these handlers only to check the numbers. Each of this Hanler should has logic which will change Storage depends on operation type